File: MobileAGECTR/ServerSide/entity/ServerResponse.php
<?php


class ServerResponse
{
    public $succes;
    public $message;

    /**
     * @return mixed
     */
    public function getSucces()
    {
        return $this->succes;
    }

    /**
     * @param mixed $succes
     */
    public function setSucces($succes)
    {
        $this->succes = $succes;
    }

    /**
     * @return mixed
     */
    public function getMessage()
    {
        return $this->message;
    }

    /**
     * @param mixed $message
     */
    public function setMessage($message)
    {
        $this->message = $message;
    }

}

File: MobileAGECTR/Android/app/src/main/java/com/cegeptr/projetagectr/logic/Entity/ServerResponse.java
package com.cegeptr.projetagectr.logic.Entity;

public class ServerResponse {

    private Boolean succes;
    private String message;

    public ServerResponse(Boolean succes, String message) {
        this.succes = succes;
        this.message = message;
    }


    public Boolean getSucces() {
        return succes;
    }

    public void setSucces(Boolean succes) {
        this.succes = succes;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }
}
